test/deployments.ts

```diff
@@ -134,6 +134,89 @@ describe("Aludel factory deployments", function () {
           const program = await factory.programs(preexistingProgram.address);
           expect(program.name).to.eq("some name");
         });
+        describe("WHEN updates the program using update-program", () => {
+          let program: AludelFactory.ProgramDataStruct;
+          beforeEach(async () => {
+            program = await factory.programs(preexistingProgram.address);
+          });
+          it("AND updates name", async () => {
+            beforeEach(async () => {
```

Comment:
somehow this *is* run, despite being a `beforeEach` inside an `it` :upside_down_face: